### ISSUE TITLE
`task export` produces invalid JSON

### ISSUE THREAD

**Issue by djmitche** [GothenburgBitFactory/taskwarrior#2037]
#### To report a bug...

* What command(s) did you run?

I have this task in my .task/pending.data:
```
[annotation_1513633823:"https:\/\/bugzilla.mozilla.org\/show_bug.cgi?id=1271677" bugzillabugid:"1271677.000000" bugzillacomponent:"General" bugzillaproduct:"Taskcluster" bugzillastatus:"NEW" bugzillasummary:"Interactive tasks should be scheduled with high priority" bugzillaurl:"https:\/\/bugzilla.mozilla.org\/show_bug.cgi?id=1271677" depends:"&open;\"371521f7-866b-4c45-a164-9b3d8edbc9c5\"&close;" description:"http:\/\/bugzil.la\/1271677 Interactive tasks should be scheduled with high priority" entry:"1513633822" modified:"1527859353" priority:"M" project:"moz" status:"waiting" tags:"bugzilla" uuid:"718d0879-b1ef-42f1-bbab-386e7d2c9608" wait:"1530451353"]
```
The `&open` in depends is due to some interaction between taskwarrior 2.5.0 and taskwarrior 2.5.1 via task sync.  But that's not the bug I'm reporting.  The problem is that task export fails to produce correct JSON for this:

```
task export 718d0879-b1ef-42f1-bbab-386e7d2c9608 | jq
parse error: Invalid numeric literal at line 2, column 325
```

The output is

```json
[
{"id":63,"bugzillabugid":1271677.000000,"bugzillacomponent":"General","bugzillaproduct":"Taskcluster","bugzillastatus":"NEW","bugzillasummary":"Interactive tasks should be scheduled with high priority","bugzillaurl":"https:\/\/bugzilla.mozilla.org\/show_bug.cgi?id=1271677","depends":["["371521f7-866b-4c45-a164-9b3d8edbc9c5"]"],"description":"http:\/\/bugzil.la\/1271677 Interactive tasks should be scheduled with high priority","entry":"20171218T215022Z","modified":"20180601T132233Z","priority":"M","project":"moz","status":"waiting","tags":["bugzilla"],"uuid":"718d0879-b1ef-42f1-bbab-386e7d2c9608","wait":"20180701T132233Z","annotations":[{"entry":"20171218T215023Z","description":"https:\/\/bugzilla.mozilla.org\/show_bug.cgi?id=1271677"}],"urgency":3.61507}
]
```

* What did you expect to happen?

valid JSON: `"depends":["[\"371521f7-866b-4c45-a164-9b3d8edbc9c5\"]"],`

* What actually happened?

invalid JSON: `"depends":["["371521f7-866b-4c45-a164-9b3d8edbc9c5"]"],`

* Paste the output of the ```task diag``` command.

```
dustin@dorp ~ $ task diag

task 2.5.0
   Platform: Linux

Compiler
    Version: 5.3.1 20160216
       Caps: +stdc +stdc_hosted +LP64 +c8 +i32 +l64 +vp64 +time_t64
 Compliance: C++11

Build Features
      CMake: 3.2.2
    libuuid: libuuid + uuid_unparse_lower
  libgnutls: 3.4.9
 Build type: None

Configuration
       File: /home/dustin/.taskrc (found), 7947 bytes, mode 100664
       Data: /home/dustin/.task (found), dir, mode 40775
    Locking: Enabled
         GC: Enabled
    $EDITOR: vim
     Server: freecinc.com:53589
         CA: /home/dustin/.task/freecinc_19447557.ca.pem, readable, 3755 bytes
      Trust: strict
Certificate: /home/dustin/.task/freecinc_19447557.cert.pem, readable, 3741 bytes
        Key: /home/dustin/.task/freecinc_19447557.key.pem, readable, 25364 bytes
    Ciphers: NORMAL
      Creds: FreeCinc/freecinc_19447557/************************************

Hooks
    Scripts: Enabled
             (-none-)

Tests
      $TERM: screen (777x214)
       Dups: Scanned 5693 tasks for duplicate UUIDs:
             Found duplicate a1205a2c-f698-438a-8493-ba458bc7173e
```

Regardless of its input, I submit that `task export` should *never* produce bad JSON data.  If there is an error in the Taskwarrior data format, that should be presented to me separately.

**Comment by ThomasAdam** [comment]
Can you confirm if this bug is present using `taskwarrior` `2.6.0` from git?  I can't seem to reproduce this.

**Comment by djmitche** [comment]
Interesting, even with 2.5.1:
```
dustin@jemison ~ $ task 3e0ea0e6-9829-4ed0-841a-3c4e9aaea984 export | jq .
[
...
    "depends": "[\"371521f7-866b-4c45-a164-9b3d8edbc9c5,04197a9b-26e2-466c-9bef-f293d266db42\"]",
```

so it seems this was fixed in 2.5.1.  I've since manually upgraded the system running 2.5.0 to 2.5.1 so hopefully I won't see this again.
